OSDOCS-5809: adds RN for NMState Operator update in  console (#63415)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

==== Update to HAProxy 2.6
With this release, {product-title} updated to HAProxy 2.6.

[id="ocp-4-14-nmstate-ui-console-update"]
==== NMstate Operator updated in console

> With this release, you can access NMstate Operator and resources from the web console. The Kubernetes NMState Operator provides a Kubernetes API for performing state-driven network configuration across the {product-title} cluster’s nodes with NMState.

Reviewer: Minor comments: 
We can access **NMstate Operator** before this release, location is k8s/ns/openshift-nmstate/operators.coreos.com~v1alpha1~ClusterServiceVersion after installing it.
As per my understanding, this new feature is focus on supporting access **NMstate resources** from the web console.

> In the *Administrator* perspective of the console to access the NMState Operator and resources, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*. See xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator] for more information and how to install the operator.

Reviewer: I believe there is another new page in this feature, Networking -> NodeNetworkState, should we also add this information in the RN?
So, for NMState resources **NNCP/NNCE**, navigate to **Networking -> NodeNetworkConfigurationPolicy**, and for NMState resource **NNS**, navigate to **Networking -> NodeNetworkState**.

==== Update to HAProxy 2.6
With this release, {product-title} updated to HAProxy 2.6.

[id="ocp-4-14-nmstate-ui-console-update"]
==== NMstate Operator updated in console

> With this release, you can now access the NMstate Operator and resources such as the `NodeNetworkState` (NNS), `NodeNetworkConfigurationPolicy` (NNCP), and `NodeNetworkConfigurationEnhancement` (NNCE) from the web console. See xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[Updating node network configuration] for more information on NMState resources and how to update them in the console.

Reviewer 2: ```suggestion
With this release, you can access the NMstate Operator and resources such as the `NodeNetworkState` (NNS), `NodeNetworkConfigurationPolicy` (NNCP), and `NodeNetworkConfigurationEnhancement` (NNCE) from the web console. For more information about NMState resources and how to update them in the console, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[Updating node network configuration].
```

In the *Administrator* perspective of the console to access the NMState Operator and resources follow these steps:

* For the NNCP, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*.

* For the NNCE, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*.

> * For the NNS, navigate to *Networking* -> *NodeNetworkState*.

Reviewer 2: Typically we don't include procedural content in RNs (the only cases I am specifically aware of are workaround steps for known issues and similar). This information seems to be covered by the procedures in the "Updating node network configuration" page linked above. Suggest deleting these lines.

Reviewer: make sense for RNs, thanks @jeana-redhat

> 
> See xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator] for more information and how to install the operator.
> 

Reviewer 2: I would also suggest removing this bit. The Operator is not new in this release (iiuc), and they will be right in that directory already from the previous link. If you do opt to keep it, would edit as follows:
```suggestion

For more information about the Operator, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator].

```

[id="ocp-4-14-nw-ipv6-spoke-cluster-support"]
==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters

> With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.

Reviewer: seems this is not related to the nmstate UI feature, should we commit this in another PR?

Author: It's trailing white space that my editor auto deletes when I save a file so builds don't break. I didn't actually write anything here or add new content.

Reviewer: yeah, okay for me, sorry for not reading carefully
